src/content.ts

```diff
@@ -464,6 +468,16 @@ function cancelSponsorSchedule(): void {
 function startSponsorSchedule(includeIntersectingSegments: boolean = false, currentTime?: number): void {
     cancelSponsorSchedule();
 
+    // Don't skip if advert playing and reset last checked time
+    if (document.getElementsByClassName('ad-showing').length > 0) {
```

Comment:
Duplicate code?

Reply:
Good point, will test if nothing breaks with isAdPlaying instead, since i was only using it for UI